README.md

```diff
@@ -26,22 +26,13 @@ The fields in the table below can be used in these parts of fiboa documents:
 
 | Property Name   | Type   | Description |
 | --------------- | ------ | ----------- |
-| ml:creation_method | string | **REQUIRED**. how the label was created e.g. field walk, image-annotated, ML generated |
-| ml:creation_method_detail | string |  additional details about the creation method, including potentially a link to the algorithm |
 | ml:boundary_type | string | **REQUIRED**. whether the boundary is for a field, ownership parcel, hedge, or other boundary |
```

Comment:
Is this meant to be an enum?

Reply:
I believe so yes, but I don't think we agreed yet on what the enum values will be. If everybody agrees on the values listed above, I can change it to enum.